### PR TITLE
Changed Leonard scaling to create non-square fault planes

### PR DESCRIPTION
**Pull request by jpa198**
Disabled broken test @baes

### CODE REVIEW

**Comment by jpa198** [comment]
Don't merge, actual testing is going badly

**Comment by jpa198** [comment]
All is well

File: srf_generation/Fault.py [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+
+    def to_dict(self):
+        base_dict: dict = super().to_dict()
+        base_dict.update({
+            "type": 3,
+        })
+        return base_dict
+
+
+class Type4(MultiPlaneFault):
```
Comment (jasonmotha) [inline srf_generation/Fault.py]: I think each plane needs an init time, but that can be an extension later.

Comment (jpa198) [reply srf_generation/Fault.py]: Yeah, Type3/4 are just an outline for when NHM faults get implemented

**Pull request merged**